data/ui_scripts/language_setting/__init__.lua

```diff
@@ -0,0 +1,153 @@
+local localization = {
+    english = {
+        LANGUAGE_BUTTON = "Language",
+        LANGUAGE_BUTTON_DESC = "Change your game language.",
+        POPUP_RESTART_REQUIRED_TITLE = "RESTART REQUIRED",
+        POPUP_RESTART_REQUIRED_TEXT = "WARNING: To work properly, you need to have your language into zone directory of your game root. You have changed your game language, wish you restart?",
```

Comment:
"... you need to have your language's zone files in the zone directory ..."

"... do you wish to restart? ..."